### ISSUE TITLE
Research: Allowing users to add new link to navigation bar

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#1432]
### Summary
@sdellis  Emma S. from SC has asked about this. She is making the Morrison DPUL exhibition and asked if it was possible to add another link to the main nav bar in DPUL.  Currently that is not an option.  What are the pros and cons to enabling this feature? 

### Impact 
_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
n/a
### Priority recommendation

- [ ] asap
- [ ] within the next 3 weeks
- [ ] PO will prioritize

## Sudden Priority Justification
_Required if "asap" or "within the next 3 weeks" is checked.  Add "Sudden Priority" and "Maintenance/Research labels_

**Comment by tpendragon** [comment]
@kelea99 Can you screenshot what the main navbar is and describe what they want to add and why?

**Comment by kelea99** [comment]
Yup! It just came in and i wanted to get a ticket made before the end of the day. Standby.

**Comment by kelea99** [comment]
You can add them vertically (via dropdown). You cannot add another navigation item horizontally. 
So you have up-to and not exceeding 4 static tabs that can be activated horizontally in the navigation bar:
1. home
2. Curated Features (with dropdowns of subpages)
3. Browse
4. About (when you click on about, you will be directed to a page and sidebar with the other subpages)
